docs/etcd-tls-assets.md

```diff
@@ -22,6 +22,34 @@ See also the [user-facing
 documentation](https://docs.openshift.com/container-platform/4.5/security/certificate-types-descriptions.html#etcd-certificates_ocp-certificates)
 for these certificates.
 
+## etcd CA summary
+
+All etcd CAs and their CA bundles are stored in the `openshift-config`
+namespace.
+
+| CA  (secrets)      | CA bundle  (configmaps)          | CA bundle also appearing in                  |
+| ------------------ | -------------------------------- | -------------------------------------------- |
+| etcd-signer        | etcd-ca-bundle                   | openshift-etcd                               |
+|                    |                                  | openshift-etcd-operator                      |
+|                    |                                  | openshift-etcd/etcd-peer-client-ca           |
+|                    | etcd-serving-ca                  | openshift-etcd                               |
+| etcd-metric-signer | etcd-metric-serving-ca           | openshift-etcd/etcd-metrics-proxy-client-ca  |
+|                    |                                  | openshift-etcd/etcd-metrics-proxy-serving-ca |
+| (bootstrap signer) | initial-etcd-signer              |                                              |
```

Comment:
This really isn't a separate CA we just reuse[1] etcd-signer based TLS assets generated by the installer.

https://github.com/openshift/cluster-etcd-operator/blob/release-4.7/pkg/cmd/render/render.go#L129

Reply:
I screwed up the name, it's `initial-etcd-ca` not `signer` (fixed). I'm not sure what you mean by this not being a separate CA - it's a ca bundle (i.e. stored in a configmap) and presumably used to validate certs issued by the installer CA. Any ideas why it still exists after install?

Reply:
gotcha yeah it was cruft ported over from KAS-O. we removed it in 4.7[1]

[1] https://github.com/openshift/cluster-etcd-operator/pull/501

Reply:
So we can remove that line then, or mark removed after 4.7?

Reply:
Done